src/components/ForestTypeComparison/index.js

```diff
@@ -1,27 +1,43 @@
-import React, { useMemo, useState } from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 import { Form, Message } from 'semantic-ui-react';
-import { info } from '@geops/tree-lib';
+import { info, sort } from '@geops/tree-lib';
 
 import Dropdown from '../Dropdown';
 import { setForestTypeComparison } from '../../store/actions';
 
 import ChForestTypeComparison from './ch';
 import LuForestTypeComparison from './lu';
 
+const getValidForestTypes = (codes, activeProfile) =>
+  codes.reduce((forestTypes, code) => {
+    try {
+      const nextFt = info('forestType', code, activeProfile);
+      return [...forestTypes, nextFt];
+    } catch {
+      return forestTypes;
+    }
+  }, []);
+
 function ForestTypeComparison() {
   const dispatch = useDispatch();
   const activeProfile = useSelector((state) => state.activeProfile);
   const codes = useSelector((state) => state.forestTypeComparison) || [];
   const [dropdownOpen, setDropdownOpen] = useState(false);
   const { i18n, t } = useTranslation();
 
-  const data = codes.map((code) => info('forestType', code, activeProfile));
+  const data = getValidForestTypes(codes, activeProfile);
+
+  useEffect(() => {
+    const validForestTypes = getValidForestTypes(codes, activeProfile);
+    dispatch(setForestTypeComparison(validForestTypes.map((ft) => ft.code)));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [activeProfile, dispatch]);
 
   const options = useMemo(
     () =>
-      info('forestType', null, activeProfile).map((ft) => ({
+      sort(info('forestType', null, activeProfile)).map((ft) => ({
```

Comment:
I would prefer something like this:
```suggestion
      info('forestType', null, activeProfile).sort((a, b) => ...).map((ft) => ({
```
In my view a `sort` function in `tree-lib` is a bit too generic.

Reply:
Done, moved the sort comparison function to src/utils